Makefile

```diff
@@ -0,0 +1,49 @@
+all: build
+.PHONY: all
+
+# Include the library makefile
+include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
+	golang.mk \
+	targets/openshift/images.mk \
+	targets/openshift/deps.mk \
+)
+
+# Exclude e2e tests from unit testing
+GO_TEST_PACKAGES :=./pkg/... ./cmd/...
+
+IMAGE_REGISTRY :=registry.svc.ci.openshift.org
```

Comment:
I'm not sure this `IMAGE_REGISTRY` is used but if it is, it should be `registry.ci.openshift.org`

Reply:
ok, thx!